tests/Transports.Subscriptions.Abstractions.Tests/Transports.Subscriptions.Abstractions.Tests.csproj

```diff
@@ -2,9 +2,6 @@
 
   <PropertyGroup>
     <TargetFrameworks>netcoreapp3.1;netcoreapp3.0;netcoreapp2.2;net471</TargetFrameworks>
-    <IsPackable>false</IsPackable>
-    <AssemblyName>GraphQL.Server.Transports.Subscriptions.Abstractions.Tests</AssemblyName>
-    <RootNamespace>GraphQL.Server.Transports.Subscriptions.Abstractions.Tests</RootNamespace>
```

Comment:
You'll want to keep the root namespace as otherwise adding files to the project will get added as the project name, which doesn't have `GraphQL.Server` at the start.

Reply:
OK